### PR TITLE
ansible: Install devtoolset-7 on cowfish

### PR DESCRIPTION
**Pull request by rleigh-codelibre**
Description is in the title.  This is exactly the same as the existing devtoolset-4 role, but for devtoolset-7.

### CODE REVIEW

File: ansible/roles/devtoolset-7/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+- name: devtoolset-7 | Install devtoolset-7
+  yum:
+    name: "{{ item }}"
+    state: latest
```
Comment (manics) [inline ansible/roles/devtoolset-7/tasks/main.yml]: This should be `present` upgrades can be handled separately.

**Comment by manics** [comment]
If the current playbooks are actually broken then perhaps it's best to remove the broken bits?

**Comment by rleigh-codelibre** [comment]
They weren't broken in the past.  Is this due to using a newer ansible version--do the roles need updating?

File: ansible/roles/devtoolset-7/tasks/main.yml (hunk 1 above)
Comment (rleigh-codelibre) [reply ansible/roles/devtoolset-7/tasks/main.yml]: I've fixed this now.

**Comment by rleigh-codelibre** [comment]
For the broken jenkinsslave playbook, looks like the files it's trying to copy like `config` and `snoopycrimecop_github` and `snoopycrimecop_github.pub` don't exist in infrastructure, management_tools or credentials.

@sbesson or @joshmoore do you know where these files are and why they aren't in git?

In the meantime, can we just run the individual playbook and avoid this defect?

**Comment by manics** [comment]
I've deployed:

```diff
diff --git a/ansible/ci-deployment.yml b/ansible/ci-deployment.yml
index 806c78e..467c0ba 100644
--- a/ansible/ci-deployment.yml
+++ b/ansible/ci-deployment.yml
@@ -43,6 +43,7 @@
   roles:
     - role: devtoolset-4
     - role: devtoolset-7
+  tags: [temp-tag]

 # Deploy Documentation prerequisites
 - hosts: ci-docs
```
`ansible-playbook -i ../../ansible/inventory/ci-hosts ci-deployment.yml --diff -l cowfish.openmicroscopy.org --ask-become-pass -t temp-tag -v`

We should figure out what's going on with the rest of the playbook.

**Comment by manics** [comment]
I've deployed the latest commit.

**Pull request closed without merging**